code/modules/surgery/blood_filter.dm

```diff
@@ -24,21 +24,21 @@
 
 /datum/surgery_step/filter_blood/preop(mob/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery)
 	display_results(user, target, "<span class='notice'>You begin filtering [target]'s blood...</span>",
-		"<span class='notice'>[user] uses the [tool] to filtering your blood.</span>",
-		"<span class='notice'>[user] uses the [tool] on [target]'s chest.</span>")
+		"<span class='notice'>[user] uses \the [tool] to filtering your blood.</span>",
+		"<span class='notice'>[user] uses \the [tool] on [target]'s chest.</span>")
```

Comment:
```suggestion
		"<span class='notice'>[user] uses [tool] to filtering your blood.</span>",
		"<span class='notice'>[user] uses [tool] on [target]'s chest.</span>")
```

`\the` is automatic.

Reply:
Is that the case for all \the 's seen above and below, or just those specifically?

Reply:
`\The` isn't implicit, but `\the` is.